### PR TITLE
Multiple items basket test

### PR DESCRIPTION
**Pull request by maninder1438**
same product with multiple quantity added and prices are checked & tested
multiple products are added and total price verified & tested

### CODE REVIEW

File: src/test/java/com/mavenit/argos/SmokeTestSuite.java [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        Thread.sleep(5000);
+        goToBasket();
+        Thread.sleep(2000);
+        String actual = getProductNameInBasket();
+        assertThat(actual, is(equalToIgnoringCase(selectedProductName)));
+
+        System.out.println("\n\n'basketTest' Test Passed - Actual selected product & the product in the basket is same");
+        System.out.println("\nTotal number of products are: " + productWebElements.size());
+        System.out.println("Index number of random product is : " + randomNumber);
+        System.out.println("\nActual selected product is: " + selectedProductName );
```
Comment (rkesireddy) [inline src/test/java/com/mavenit/argos/SmokeTestSuite.java:91]: Code should not contains System.out

File: src/test/java/com/mavenit/argos/SmokeTestSuite.java [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    public void basketMultipleQuantityTest() throws InterruptedException {
+
+        doSearch("tripods, monopods and cases");
+        Thread.sleep(4000);
+        List<WebElement> productWebElements = driver.findElements(By.cssSelector("a[data-test='component-product-card-title']"));
+        List<WebElement> price = driver.findElements(By.cssSelector("div[class^=\"ProductCardstyles__PriceText-l8f8q8-14\"]"));
+        if (productWebElements.size() == 0) {
+            Assert.fail("No Products found with: " + "tripods, monopods and cases");
+        }
+        // TODO: 2020-02-08 this will be converted in future
```
Comment (rkesireddy) [inline src/test/java/com/mavenit/argos/SmokeTestSuite.java]: Remove random class from here and make it separate class like helpers class, as I can see u using random more than 1 and in future you will be using more

File: src/test/java/com/mavenit/argos/SmokeTestSuite.java [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+        System.out.println("Index number of random product is : " + randomNumber);
+
+        System.out.println("\nActual selected product is: " + selectedProductName );
+        System.out.println("Product in the Basket is:   " + actualName);
+    }
+
+
+    @Test
+    public void basketMultipleProductTest() throws InterruptedException {
+        int numberOfProductsBuying = 2;
```
Comment (rkesireddy) [inline src/test/java/com/mavenit/argos/SmokeTestSuite.java]: 2 is really small number , don’t you thing it will waste memory by storing in Int data type

Comment (maninder1438) [reply src/test/java/com/mavenit/argos/SmokeTestSuite.java]: my mistake,, now data type has been changed to byte,, in build and working successfully as expected

File: src/test/java/com/mavenit/argos/SmokeTestSuite.java (hunk 2 above)
Comment (maninder1438) [reply src/test/java/com/mavenit/argos/SmokeTestSuite.java]: done, i made separate random class and whenever i need this class i will call it

File: src/test/java/com/mavenit/argos/SmokeTestSuite.java (hunk 1 above)
Comment (maninder1438) [reply src/test/java/com/mavenit/argos/SmokeTestSuite.java:91]: i was trying to parametrised this sout but unbale to do it. just using sout again and again to export the results on external result.txt file. i need your help in this in next session

**Pull request merged**